packages/path-provider/pubspec.yaml

```diff
@@ -1,5 +1,9 @@
 name: path_provider
+
+version: 0.1.0
 description: Flutter plugin for finding commonly used locations on the filesystem.
```

Comment:
s/Flutter plugin/a Flutter plugin/ for consistency with all the existing plugins on pub